Laravel-whib/app/Http/Controllers/MapController.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\MapPin;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Validation\ValidationException;
+
+class MapController extends Controller
+{
+    /**
+     * Display a listing of the map pins.
+     *
+     * @return \Illuminate\Http\JsonResponse
+     */
+    public function index()
```

Comment:
I think this method won't be usefull in controller. The method we need is to get all user pins and user pins by categories and names.